ccd-definition/FixedLists/OrderType-SDO-nonprod.json

```diff
@@ -2,7 +2,7 @@
   {
     "ID": "OrderType",
     "ListElementCode": "DISPOSAL",
-    "ListElement": "Disposal/Final hearing",
+    "ListElement": "Disposal hearing",
```

Comment:
This change seems to covered in your other PR: https://github.com/hmcts/civil-ccd-definition/pull/675/files#diff-667cf579b50407b335e4472776db8223902dbf44bc94747f0714998e7288e1acR16

you could remove this PR or remove the change from your other PR, up to you